dbt_subprojects/tokens/models/transfers_and_balances/gnosis/tokens_gnosis_suicide_transfers.sql

```diff
@@ -0,0 +1,134 @@
+{{config(
+    schema = 'tokens_gnosis',
+    alias = 'suicide_transfers',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+    unique_key = ['block_date','unique_key'],
+)
+}}
+
+WITH 
+
+suicide AS (
+    SELECT 
+         cast(date_trunc('day', block_time) as date) AS block_date
+        , block_time
+        , block_number
+        , tx_hash
+        , tx_index
+        , trace_address
+        , tx_from
+        , tx_to
+        , address 
+        , refund_address 
+    FROM 
+        {{ source('gnosis', 'traces') }}
+    WHERE
+        type = 'suicide'
+        AND
+        success
+),
+
+suicide_events AS (
+    SELECT
+        *
+        ,ROW_NUMBER() OVER (PARTITION BY address ORDER BY block_time) AS event_sequence
+        ,LAG(block_time) OVER (PARTITION BY address ORDER BY block_time) AS previous_block_time
+    FROM
+        suicide
+),
+
+tokens_gnosis_base_without_suicide_transfers AS (
+    SELECT
+        address
+        ,event_sequence
+        ,SUM(amount_raw) AS amount_raw
+    FROM (
+        SELECT
+            t2.address 
+            ,-t1.amount_raw AS amount_raw
+            ,t2.event_sequence
+        FROM    
+            {{ ref('tokens_gnosis_base_without_suicide_transfers') }} t1
```

Comment:
same question on this one, if the above adds incremental, we should here too to match